test.sh

```diff
@@ -6,6 +6,8 @@ set -eux
 HOST=http://localhost:8420
 SK=sk_test_12345
 
+curl -X DELETE $HOST/_config/data
+
```

Comment:
Because it's not related to a localstripe feature, but it's more a CI change, this change needs to be inside another PR.
Like that we would be able to discuss it, as long as needed, without blocking this PR.

Reply:
👍 removed this line